meson/post_install.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+from os import path, environ
+import subprocess
+
+prefix = environ.get('MESON_INSTALL_PREFIX', '/usr/local')
```

Comment:
What if the user installs to `/usr`?

Reply:
I referred to elementary existing apps too. I think apps are supposed to be installed in this directory.